hpx/lcos/local/promise.hpp

```diff
@@ -253,16 +287,24 @@ namespace hpx { namespace lcos { namespace local
 
         // Returns: A future<R> object with the same shared state as *this.
         // Throws: future_error if *this has no shared state or if get_future
-        //         has already been called on a promise with the same shared
-        //         state as *this.
+        //         or get_shared_future has already been called on a promise
```

Comment:
These don't of course show up in docs because it's not in doxygen format. Probably better to leave it for another PR though because there are plenty of other places where we have this kind of almost-doxygen docs.

Reply:
I agree. I simply went with what we have already in the same file.